assets/css/style.css

```diff
@@ -87,3 +119,89 @@ button {
 .error {
 	color: red;
 }
+
+
+/* profile */
+
+.upload-form-container {
+	margin: 50px;
+}
+
+
+.upload-form {
+ margin-right: 25%;
+ margin-left: 25%;
+}
+
+.upload-button {
+	background-image: url("upload.png");
```

Comment:
This needs to be a full path to `upload.png`, not just the name. How are other images referenced in here?